README.md

```diff
@@ -39,16 +38,36 @@ with only a small set of [external requirements](#requirements) needed.
 
 Packages are installed in Neovim's data directory ([`:h standard-path`][help-standard-path]) by default. Executables are
 linked to a single `bin/` directory, which `mason.nvim` will add to Neovim's PATH during setup, allowing seamless access
-from Neovim builtins (shell, terminal, etc.) as well as other 3rd party plugins.
+from Neovim builtins (LSP client, shell, terminal, etc.) as well as other 3rd party plugins.
 
 For a list of all available packages, see <https://mason-registry.dev/registry/list>.
 
-## Screenshots
+# Installation & Usage
 
-|                                                                                                                                                        |                                                                                                                                                  |                                                                                                                                        |
-| :----------------------------------------------------------------------------------------------------------------------------------------------------: | :----------------------------------------------------------------------------------------------------------------------------------------------: | :------------------------------------------------------------------------------------------------------------------------------------: |
-|           <img alt="Main window" src="https://user-images.githubusercontent.com/6705160/177617680-d62caf26-f253-4ace-ab57-4b590595adca.png">           |                 <img src="https://user-images.githubusercontent.com/6705160/177617684-6bb4c13f-1235-4ac9-829e-120b06f7437b.png">                 | <img alt="Language filter" src="https://user-images.githubusercontent.com/6705160/177617688-8f9ba225-00c8-495c-9c4c-b74240d6f280.png"> |
-| <img alt="LSP server configuration schema" src="https://user-images.githubusercontent.com/6705160/177617692-02c6ddde-a97e-42b4-bca4-4f4caf45d569.png"> | <img alt="Checking for new versions" src="https://user-images.githubusercontent.com/6705160/180648183-69077d10-8795-4da6-ba4d-57ecf0cb25c9.png"> |   <img alt="Help window" src="https://user-images.githubusercontent.com/6705160/180648292-136a0888-0fb6-4226-aa29-53bd3ffed400.png">   |
+> [`:h mason-quickstart`][help-mason-quickstart]
+
+Install using your plugin manager of choice. **Setup is required**:
+
+```lua
+require("mason").setup()
+```
+
+`mason.nvim` is optimized to load as little as possible during setup. Lazy-loading the plugin, or somehow deferring the
+setup, is not recommended.
+
+Refer to the [Configuration](#configuration) section for information about which settings are available.
+
+## Recommended setup for `lazy.nvim`
+
+The following is the recommended setup when using `lazy.nvim`. It will set up the plugin for you, meaning **you don't have
+to call `require("mason").setup()` yourself**.
```

Comment:
Is calling`setup` multiple times an issue for mason? If so, this could simply be fixed programmatically, by setting a `masonWasAlreadySetup` variable that disallows running `setup` twice. mason is one of the first plugins new users install, so keeping it as fool-proof as possible might make sense.

Alternatively, mason could also support changing the after when `setup` is called. Though that's not a hard standard, it is a pretty common practice among (newer) nvim plugins to allow the user to change a config by running `setup` again. Though this take a bit more effort, so disallowing multiple setup calls might be the simpler approach.

Reply:
Then how about using `has_setup` in `mason.nvim` as well to prevent multiple calls? Would be the simplest solution